.gitignore

```diff
@@ -161,4 +161,6 @@ cython_debug/
 
 **/env/
 **/key.txt
-**/local-auth.json
\ No newline at end of file
+**/local-auth.json
+tempo/
```

Comment:
The `tempo` folder and the `works.ipynb` file are not a part of the main project. Please, replace this.